colin/core/checks/abstract_check.py

```diff
@@ -33,15 +33,19 @@ def check(self, target):
         pass
 
     def __str__(self):
-        return "{}\n" \
-               "   -> {}\n" \
-               "   -> {}\n" \
-               "   -> {}\n" \
-               "   -> {}\n".format(self.name,
-                                   self.message,
-                                   self.description,
-                                   self.reference_url,
-                                   ", ".join(self.tags))
+        return (
+            "{}\n"
+            "   -> {}\n"
+            "   -> {}\n"
+            "   -> {}\n"
+            "   -> {}\n".format(
+                self.name,
+                self.message,
+                self.description,
+                self.reference_url,
+                ", ".join(self.tags),
```

Comment:
We already depends on python-3.6

Should we use f-string?

It can be done in different PR

Reply:
Nice! I'll create a followup for fstrings, format...

Reply:
BTW we needn't use f-string everywhere but IMHO it would simplify some cases.